docs/book/src/reference/envtest.md

```diff
@@ -36,7 +36,7 @@ test: manifests generate fmt vet
 Or configure the existing target to skip the download and point to a [custom location](#environment-variables):
 
 ```sh
-make test SKIP_FETCH_TOOLS=1 KUBEBUILDER_ASSETS=/opt/kubebuilder/testbin
+make test SKIP_FETCH_TOOLS=1 KUBEBUILDER_ASSETS=/usr/local/kubebuilder
```

Comment:
I think the whole section on configuring the envtest binaries is outdated, since we now use the `setup-envtest` [tool](https://pkg.go.dev/sigs.k8s.io/controller-runtime/tools/setup-envtest). We should probably rewrite the entire section instead of making this modification to the path? Since with the recent changes we do not have a separate envtest script, and the location of the binaries can be specified with a separate flag passed to `setup-envtest` tool.

Reply:
@varshaprasad96 @rashmigottipati The docs are a bit outdated. There is a related issue I'm working on, as well #2330.

`kubebuilder@v3.1.0` uses `controller-runtime@v0.8.3` which isn't yet updated to the `setup-envtest` tool. Should the docs reflect the most recent release or the most recent commit to the default branch? I'm happy to open PRs against specific branches to align them with the state of the world at that time, but I don't know that multiple versions of the `kubebuilder` book would be helpful.

Regardless, updating the docs to reflect the new tooling would be incorrect with the most recent release. If the book is deployed from the most recent release, then a change to a release branch in tandem with a change to the main branch would be appropriate.

Reply:
Hi @Birdrock, 

Did you test run the make test command locally with your changes? if yes, could you add the result of the test in the description of this PR? That helps us to review it. 

Also, I agree with @varshaprasad96 we need to update this doc to add the new way to set up the env tests. The master branch should reflect the state of the master code and the changes in the doc will be in prod when the code source be releassed. 

WDYT? Could you work on that and help us with it?